### DISCUSSION TITLE
新增关于自动降落的测试功能，请不要随意开启

### DISCUSSION THREAD

**Discussion by GlaringMarvel** [GlaringMarvel/Aokana-Flying-Circus#7] [Announcements]
请不要在训练房以外使用自动降落功能，这是我用来测试的
这可能会是最后一个大功能增加，以后就只剩修修bug加加小功能了
脚本的大部分功能开发就到此为止了，有bug可以继续向我反馈，缝缝补补还是会做的

<hr /><em>This discussion was created from the release <a href='https://github.com/GlaringMarvel/Aokana-Flying-Circus/releases/tag/v1.1.3'>新增关于自动降落的测试功能，请不要随意开启</a>.</em>

**Comment by OctoberCG** [comment]
有没有海战脚本分享
